packages/waku/src/router/create-pages.ts

```diff
@@ -30,6 +30,27 @@ const sanitizeSlug = (slug: string) =>
 
 // createPages API (a wrapper around unstable_defineRouter)
 
+/**
+ * Type version of `String.prototype.split()`. Splits the first string argument by the second string argument
+ * @example
+ * ```ts
+ * // ['a', 'b', 'c']
+ * type Case1 = Split<'abc', ''>
+ * // ['a', 'b', 'c']
+ * type Case2 = Split<'a,b,c', ','>
+ * ```
+ */
+export type Split<
```

Comment:
I would like to not export types until it's necessary.

```suggestion
type Split<
```

Reply:
> Other than that

I forgot to submit this comment previously...

Reply:
fixed

Reply:
Sorry, I wasn't very clear. Can you do it for all new types?

Reply:
oh, yes! Done!